src/test/java/com/microsoft/jenkins/appservice/commands/FileDeployCommandTest.java

```diff
@@ -24,9 +24,11 @@
 import static org.mockito.ArgumentMatchers.anyString;
 import static org.mockito.ArgumentMatchers.eq;
 import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.never;
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.doThrow;
```

Comment:
seems not sorted. is checkstyle enabled? if  no, pls refer to https://github.com/Microsoft/azure-spring-boot/tree/master/config

Reply:
fixed temporarily. Will enable checkstyle for test codes in another PR